relays/bin-substrate/src/cli/init_bridge.rs

```diff
@@ -108,9 +108,9 @@ impl InitBridge {
 	/// Run the command.
 	pub async fn run(self) -> anyhow::Result<()> {
 		select_bridge!(self.bridge, {
-			let source_client = self.source.into_client::<Source>().await?;
-			let target_client = self.target.into_client::<Target>().await?;
-			let target_sign = self.target_sign.into_keypair::<Target>()?;
+			let source_client = self.source.to_client::<Source>().await?;
+			let target_client = self.target.to_client::<Target>().await?;
```

Comment:
Does the sequence of `soruce_client` and `target_client` actually matter here?

Reply:
Kind of - it will affect the errors you get when we can't connect to the nodes, but overall no - it does not matter.